esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -372,23 +372,20 @@ define([
                 DropZoneFolders: true
             });
 
+            this.filter.setValue(this.id + "MaxNumberOfFiles", 1000);
+            this.filter.emit("apply");
             this.initWorkunitsGrid();
 
             this.filter.on("clear", function (evt) {
                 context.refreshGrid();
             });
             this.filter.on("apply", function (evt) {
-                context.refreshGrid();
-            });
-            topic.subscribe("hpcc/dfu_wu_completed", function (topic) {
-                context.refreshGrid();
-            });
-
-            WsDfu.DFUQuery({
-                request: {}
-            }).then(function (response) {
-                if (lang.exists("DFUQueryResponse.Warning", response)) {
-                    if (response.DFUQueryResponse.Warning) {
+                var retVal = context.filter.toObject();
+                WsDfu.DFUQuery({
+                    request: {}
+                }).then(function (response) {
+                if (lang.exists("DFUQueryResponse.DFULogicalFiles.DFULogicalFile", response)) {
+                    if (response.DFUQueryResponse.NumFiles > 100000 && !retVal.MaxNumberOfFiles) {
```

Comment:
Yes. https://track.hpccsystems.com/browse/HPCC-17164 I will change logic to check for DFUQueryResponse.Warning then since the new response structure will only show warning when user requests too many because of filter submission.